index.js

```diff
@@ -61,7 +62,13 @@ module.exports = function (version, map) {
           })
         }
       })
-    })
+    }
+
+    if (process.title == 'browser') {
+      // in browser level is stored inside IndexedDB
+      dirReady()
+    } else
+      mkdirp(path.join(dir, name), dirReady)
```

Comment:
mkdirp isn't necessary for LevelDB either

Reply:
do you mean leveldb will handle that? what if the dir is `/tmp/foo/bar/baz` and only `/tmp` exists? iirc leveldb creates one directory, but not nested directories

Reply:
Just tested it and found there's a difference between platforms: on windows it does create directories recursively, on linux it doesn't. TIL

Reply:
haha, yeah that's why this is here!